src/lib/__tests__/runSynthetics.test.ts

```diff
@@ -58,6 +58,17 @@ describe("The runSynthetics function", () => {
     expect(startAutomatedTestsSpy).toBeCalledTimes(6);
   });
 
+  it("should catch and log an error that was thrown when no batch id was returned", async () => {
```

Comment:
Do you want to add a test for a fully fleshed out return object

Reply:
I'll add to the other ones, good call